src/app/docs/split-view/split-view-docs.component.ts

```diff
@@ -2,8 +2,162 @@ import {
   Component
 } from '@angular/core';
 
+import {
+  FormControl,
+  FormGroup
+} from '@angular/forms';
+
+import {
+  Subject
+} from 'rxjs';
+
+import {
+  SkyConfirmCloseEventArgs,
+  SkyConfirmService,
+  SkyConfirmType
+} from '@skyux/modals';
+
+import {
+  SkySplitViewMessage,
+  SkySplitViewMessageType
+} from '@skyux/split-view';
+
 @Component({
   selector: 'app-split-view-docs',
   templateUrl: './split-view-docs.component.html'
 })
-export class SplitViewDocsComponent { }
+export class SplitViewDocsComponent {
+
+  public set activeIndex(value: number) {
+    this._activeIndex = value;
+    this.activeRecord = this.items[this._activeIndex];
+    this.loadFormGroup(this.activeRecord);
+  }
+
+  public get activeIndex(): number {
+    return this._activeIndex;
+  }
+
+  public activeRecord: any;
+
+  public items = [
+    {
+      id: 1,
+      amount: 73.19,
+      date: '5/13/2019',
```

Comment:
Not a huge deal, but since we're already porting this demo over, I'd recommend updating the dates from 2019 to 2020, both in the date fields and in the receipt image names. Updating this isn't essential, but it'll make the demo seem less out of date and extend the lifetime before it feels out of date.